README.md

```diff
@@ -27,6 +27,63 @@ The standalone runtime is also deployed to https://code-dot-org.github.io/ml-pla
+- it allows for the exploration of columns and their properties before they are selected.
+
+When selecting a label, the properties panel on the right shows information for the highlighted column:
+- a bar graph for categorical data;
+- min/max/range for numerical data.
+
+When selecting a feature, it shows the above properties for the highlighted column, but also attempts to show information about the relationship between that column and the previously-selected label:
+- a custom CrossTab when both label and feature are categorical data;
+- a scatterplot when both label and feature are numerical data.
+
+For student-uploaded CSV files, we attempt to guess the column type, but provide a way for the student to override this.
+
+#### Train Model
+A.I., the bot from AI for Oceans, is back.  Here, we attempt to illustrate the training process, in which a large amount of the original tabular data is given to the bot, row by row.  The animation is done by updating the React state 30 times a second to trigger a re-render each time.  The scene supports very small datasets, finishing early, as well as larger datasets, fading out after showing a few iterations of the animation.
+
+#### Generate Results
+In his scene we attempt to show A.I. generating results, again inspired by the way we presented it in AI for Oceans.  Here, the bot is making predictions on rows that were reserved for this purpose: it takes the features and predicts the label for each of those rows.  As in the previous scene, it handles both very small and larger datasets.
+
+#### Results
+This screen does a few things:
+- It shows the result for the most recent set of predictions.  That is, it shows the statement ("Predict [feature] based on [labels].") along with the accuracy (which is how well it predicted labels in the previous scene, when comparing those labels against their actual values, since this prediction was done using a reserved set of data from the original tabular data).
+- It shows the results of previous sets of predictions, with their respective statements.  This way the student can compare statements to see which have the highest "predictive power".
+- It lets the student view details of the most recent set of predictions, which shows in a pop-up.  This view has a toggle between showing correct and incorrect predictions.  In the table, the student can examine each row of reserved data to see how the label's actual value compares to what was predicted.
+- The student can "Try it out!" and run their own predictions.  The interface here is very similar to what they can see in App Lab once they import the saved model there.  The A.I. bot makes a reappearance here, because it's popular!
+
+#### Save Model
+
+The student can fill out the "model card" and then save the model, along with the model card information, to the server.  This model can then be imported in App Lab.  The model card information is seen in App Lab when previewing the model prior to importing it.
```

Comment:
I wonder about a sentence in here on the importance of Model Cards to the curriculum to justify why we included this step.